main/lint-baseline.xml

```diff
@@ -0,0 +1,1590 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="5" by="lint 4.1.2">
+
+    <issue
+        id="MissingSuperCall"
+        severity="Error"
+        message="Overriding method should call `super.onCreate`"
+        category="Correctness"
+        priority="9"
+        summary="Missing Super Call"
+        explanation="Some methods, such as `View#onDetachedFromWindow`, require that you also call the super implementation as part of your method."
+        errorLine1="    public void onCreate(final Bundle savedInstanceState) {"
+        errorLine2="                ~~~~~~~~">
+        <location
+            file="C:\private\geocaching\github\cgeo\main\src\cgeo\geocaching\AboutActivity.java"
```

Comment:
Will it work with these absolute pathes?

Reply:
Hehe, didn't see that this is in the files. So you now know my local folder structure. I will probably replace the "root".

I just took the (locally created) lint xml report files and removed all "Warnings" from it. I think there's far more in this report file than really needed for the "lint-baseline" check, this line among them. But it is helpful I guess for those later trying to solve these issues to have this information.

Whether it will work with these paths: we will see it soon in this PR. If Jenkins can build it then it works (assuming that it does not have the very same file structure :-)).

Reply:
Jup, does not work, have to replace with relative paths. Done.